### PR TITLE
Editorial changes 

### PR DESCRIPTION
**Pull request by lvdbrink**
As a result of my readthrough of the spec, as requested in #358, here are some editorial changes for the consideration of the SWG.

### CODE REVIEW

File: 1.1/spec/11-Part-08.adoc [hunk 1]
```diff
@@ -666,9 +666,9 @@ The function http://www.opengis.net/def/function/geosparql/asKML[`geof:asKML`] c
 
 ==== Discrete Global Grid System
 
-This section establishes the Requirements for representing Discrete Global Grid System (DGGS) Geometry data as RDF literals. The form of representation is specific to individual DGGS implementations: known DGGSes are not compatible or even very similar. 
+This section establishes the Requirements class for representing Discrete Global Grid System (DGGS) Geometry data as RDF literals. The form of representation is specific to individual DGGS implementations: known DGGSes are not compatible or even very similar. 
 
-Here is defined one RDFS Datatypes:
+T,he Requirements class defines one RDFS Datatype
```
Comment (nicholascar) [inline 1.1/spec/11-Part-08.adoc:671]: Typo in edit?

**Review by nicholascar — APPROVED** [review]
Thanks for these great edits @lvdbrink. 

It seems that not only is your secondary English better than my secondary Dutch (100% better) but perhaps your secondary English is better than my native English too...!

File: 1.1/spec/11-Part-08.adoc (hunk 1 above)
Comment (lvdbrink) [reply 1.1/spec/11-Part-08.adoc]: Yes, well spotted! Now corrected.

File: 1.1/spec/05-Part-02.adoc [hunk 2]
```diff
@@ -21,7 +21,7 @@ Many Conformance Classes are parameterized. For them, the list of parameters is
 | Query Rewrite Extension | Defines query transformation rules for computing spatial relations between spatial objects based on their associated geometries | A.6
 |===
 
-Dependencies between each GeoSPARQL Conformance Class are shown above in Figure 2. To support a Conformance Class for a given set of parameter values, an implementation must support each dependent Conformance Class with the same set of parameter values.
+Dependencies between each GeoSPARQL Conformance Class are shown below in Figure 2. To support a Conformance Class for a given set of parameter values, an implementation must support each dependent Conformance Class with the same set of parameter values.
```
Comment (dr-shorthair) [inline 1.1/spec/05-Part-02.adoc:24]: Maybe delete 'below' in case the figure gets shifted in formatting?

File: 1.1/spec/19-Annex-D.adoc [hunk 3]
```diff
@@ -40,7 +39,7 @@ In particular, the following attributes of the graph are validated:
 
 |http://www.opengis.net/def/geosparql/validator/S1-a-hasGeometry-hasSerialization-sub[Shape 1a]
 | Violation
-| Each node with an incoming <<Property: geo:hasGeometry, geo:hasGeometry>>, or a specialization of it, should have minimum one outgoing relation that is either <<Property: geo:hasSerialization, geo:hasSerialization>>, or a specialization of it.
+| Each node with an incoming <<Property: geo:hasGeometry, geo:hasGeometry>>, or a specialization of it, should have at minimum one outgoing relation that is either <<Property: geo:hasSerialization, geo:hasSerialization>>, or a specialization of it.
```
Comment (dr-shorthair) [inline 1.1/spec/19-Annex-D.adoc:42]: 'have a minimum of one' ?

**Review by dr-shorthair — APPROVED** [review]
I made two tiny suggestions in separate comments

**Pull request merged**